packages/front-end/components/Experiment/MetricsSelector.tsx

```diff
@@ -291,7 +295,7 @@ const MetricsSelector: FC<{
       helpText={
         <>
           {helpText}
-          {showMetricGroupHelper ? (
+          {showMetricGroupHelper && datasource ? (
```

Comment:
question (nit): Why are we checking `datasource` explicitly here in addition of the check that happens in `showMetricGroupHelper`?

Reply:
Typescript was complaining.  Before, it was smart enough to know that if `showMetricGroupHelper` was true, then datasource was definitely defined.  But my change confused Typescript.